_posts/2019-01-09-tools-we-use.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title:  "Tools We Use at Ombu Labs"
+date: 2019-01-09 10:00:00
+categories: ["agile"]
+author: "emily"
+---
+
+At Ombu Labs, we have a number of tools that help us to keep organized, write better code and communicate more effectively with our clients.
+
+<!--more-->
+
+### Daily Work
+- We use [Slack](https://ombulabs.slack.com/) to chat with team members while working.
```

Comment:
@ercohen14 Maybe it would be better to put a link to https://slack.com and also mention that we use Slack to communicate with the client's team members? We prefer this over email.